src/Form/EntityEmbedDialog.php

```diff
@@ -369,10 +368,6 @@ public function buildEmbedStep(array $form, FormStateInterface $form_state) {
       '#type' => 'hidden',
       '#value' => $entity_element['data-entity-type'],
     );
-    $form['attributes']['data-entity-id'] = array(
-      '#type' => 'hidden',
-      '#value' => $entity_element['data-entity-id'],
-    );
     $form['attributes']['data-entity-uuid'] = array(
       '#type' => 'hidden',
       '#value' => $entity_element['data-entity-uuid'],
```

Comment:
This is a hidden form element. It gets its value not from a validate or submit callback. It gets it from `$entity_element`. Which I guess works because `$entity_element` is recalculated every time an auto-complete is submitted?

If so, please document this.

However, I still wonder if this _actually_ works when you start to link to an entity? (Or perhaps my mention of autocompletes is completely off even, I've been looking at LinkIt lately, so my memory of `entity_embed` is probably warped.)